pkg/pillar/cmd/zedkube/etherpassthrough.go

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2024 Zededa, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//go:build kubevirt
+
+package zedkube
+
+import (
+	"fmt"
+
+	"github.com/lf-edge/eve/pkg/pillar/kubeapi"
+	"github.com/lf-edge/eve/pkg/pillar/types"
+	"github.com/vishvananda/netlink"
+)
+
+// checkIoAdapterEthernet - check and create NAD for direct-attached ethernet
+func checkIoAdapterEthernet(ctx *zedkubeContext, aiConfig *types.AppInstanceConfig) error {
+
+	if aiConfig.FixedResources.VirtualizationMode != types.NOHYPER {
+		return nil
+	}
+	ioAdapter := aiConfig.IoAdapterList
+	for _, io := range ioAdapter {
+		if io.Type == types.IoNetEth {
+			nadname := "host-" + io.Name
+			_, ok := ctx.networkInstanceStatusMap.Load(nadname)
+			if !ok {
+				bringupInterface(io.Name)
+				err := ioEtherCreate(ctx, &io)
+				if err != nil {
+					log.Errorf("checkIoAdapterEthernet: create io adapter error %v", err)
```

Comment:
```suggestion
					log.Errorf("checkIoAdapterEthernet: create io adapter error %v", err)
					continue
```
if it is an error, why store the `nadname`?

Reply:
I think in here, there is a case the NAD already exist on the kubernetes side, we want to count this as created but log an error.

Reply:
in another outstanding PR https://github.com/lf-edge/eve/pull/3841, we have added the function 'func CheckEtherPassThroughNAD(nadName string)', we can later to add a check to see if it already exists condition if not, then do the 'continue' as suggested.